src/components/_dialog.scss

```
@@ -0,0 +1,212 @@
+/*doc
+---
+title: Dialog
+name: dialog
+category: Components
+---
+
+ページ中央に浮遊して現れるダイアログウインドウを生成するコンポーネントです。`body`, `footer`の２要素があります。
+「閉じる」ボタンやタイトル、アクションボタン等を使用したい場合はApp Barコンポーネントを使用してください。
+
+ダイアログの領域外を暗くする場合や、領域外を選択してダイアログを閉じるアクションを実装する場合は、`ncgr-overlay`を併せて使用してください。
+また、`.ncgr-dialog`をアクティブにするときには、同時にbodyに`.overflow-hidden`を付与してください。
+下の要素のスクロールを制限し、モーダル内のスクロールの誤動作を防ぐことができます。
+
+```code
+<div class="ncgr-overlay -active">
+</div>
+<div class="ncgr-dialog -active">
+  <div class="ncgr-app-bar -active">
+    <div class="ncgr-app-bar__leading">
+      <div class="ncgr-app-bar__icon ncgr-typography-label">
+        ×
+      </div>
+    </div>
+    <div class="ncgr-app-bar__title">
+      甲賀流
+    </div>
+    <div class="ncgr-app-bar__actions">
+      <li>
+        <i class="fas fa-ellipsis-v ncgr-typography-label"></i>
+      </li>
+    </div>
+  </div>
+  <div class="ncgr-dialog__body">
+    <div class="ncgr-pad-16">
+      甲賀流（こうかりゅう）とは、近江国甲賀の地に伝わっていた忍術流派の総称。<br>
+      山を一つ隔てた場所に存在する伊賀流と並び、最も有名な忍術の一派として知られる。<br>
+      甲賀流伊賀流ともに伝系に関しては仮託の説が多く信用できるものはあまりない。<br>
+      しかし、民家で史料が発見されるなど研究も進み、少なくとも江戸時代には確実に存在したとされている。<br>
+      なお、「甲賀流」という名称の単一の流派は存在せず、あくまで甲賀に伝わる複数の流派があわさって甲賀流と呼ばれているにすぎない。<br>
+    </div>
+  </div>
+  <div class="ncgr-dialog__footer">
+    <button class="ncgr-button ncgr-button-default ncgr-button--fluid ncgr-button--light">とじる</button>
+  </div>
+</div>
+```
+
+*/
+
+.ncgr-dialog {
+  width: 40rem;
+  height: auto;
+  max-height: calc(100vh - 12rem);
+  position: fixed;
+  top: 50vh;
+  left: calc(50vw - 20rem);
+  transform: translateY(-50%);
+  background-color: var(--color-secondary-grouped-background);
+  border-radius: 5px;
```

4の倍数で設計したいので、0.5rem で。

```suggestion
  border-radius: 0.5rem;
```

---

border-radiusとかの数字は固定だとばかり思っていました！